AIOFabricator/AiOFab.cs

```diff
@@ -70,7 +73,8 @@ private void RegisterCraftTreeBasics()
 #if SUBNAUTICA
             RegisterTopLevelVanillaTab(CyclopsFabScheme, "Cyclops Upgrades", TechType.Cyclops);
 #elif BELOWZERO
-            RegisterTopLevelVanillaTab(SeaTruckFabScheme, "SeaTruck Fabricator", TechType.SeaTruck);
+            // The Seatruck Fab is basically a copy of the basic fab, so let's just ignore it.
+            //RegisterTopLevelVanillaTab(SeaTruckFabScheme, "SeaTruck Fabricator", TechType.SeaTruck);
```

Comment:
In this case I would have actually said ignore the regular and put in the seatruck as there are some thing I think in the seatruck that are not in the regular.